### PR TITLE
Expand comment box in task pages to fit whole grid

### PR DESCRIPTION
**Pull request by svineet**
Task: http://www.google-melange.com/gci/task/view/google/gci2014/5800321825636352

### CODE REVIEW

File: static/themes/SSF/style.css [hunk 1]
```diff
@@ -46,3 +46,9 @@
     background-image: url(../../img/sunflower_spin_throbber.gif);
     background-repeat: no-repeat;
 }
+
+/* Comments */
+
+#comment-form {
+    width: 100%;
```
Comment (hitesh96db) [inline static/themes/SSF/style.css:53]: Did you reduce the screen size and check ?
Maybe this could be 50 % ?
IMO 100% doesn't look good.

Comment (hitesh96db) [inline static/themes/SSF/style.css:53]: I don't think that this is a good ticket to work on...not really a bug

Comment (svineet) [inline static/themes/SSF/style.css:53]: @hitesh96db can I work on this? http://sunflower.sahanafoundation.org/eden/project/task/278

100% is looking good in smaller width screens. Should I change to 50%?

Comment (hitesh96db) [inline static/themes/SSF/style.css:53]: Not sure about eden tests...
but if you want something to work on:
try this  http://sunflower.sahanafoundation.org/eden/project/task/389 ?

Comment (svineet) [inline static/themes/SSF/style.css:53]: @hitesh96db I don't think I will be able to do that... Can you give me something of difficulty similar to my previous ticket "My Task"?

Comment (svineet) [inline static/themes/SSF/style.css:53]: @hitesh96db maybe there should be separate tasks for adding test cases?

**Pull request closed without merging**